lightning_training_studio/controllers/sweep.py

```diff
@@ -47,11 +48,12 @@ def on_reconcile_start(self, sweeps: List[SweepConfig]):
                 if id not in self.tensorboard_sweep_id:
                     self.tensorboard_sweep_id.append(id)
                     self.db.insert(TensorboardConfig(sweep_id=id, shared_folder=str(drive.drive_root)))
-                elif sweep.stage in (Stage.FAILED, Stage.SUCCEEDED):
+                elif sweep.stage in (Stage.FAILED, Stage.SUCCEEDED) and id not in self.tensorboard_sweep_id_detach:
                     for tensorboard in self.db.select_all(TensorboardConfig):
                         if tensorboard.sweep_id == id:
-                            tensorboard.desired_stage = Stage.DELETED
+                            tensorboard.desired_stage = Stage.STOPPED
                             self.db.update(tensorboard)
+                            self.tensorboard_sweep_id_detach.append(id)
```

Comment:
Fix a bug where the Sweep Controller would keep stopping the Tensorboard instance.